js/src/ui/Actionbar/Search/search.css

```diff
@@ -27,6 +27,18 @@
   width: 500px !important;
 }
 
```

Comment:
For reference purposes: https://github.com/TeamWertarbyte/material-ui-chip-input/issues/20 may be relevant.